examples/nextjs/src/components/my-page.js

```diff
@@ -25,9 +24,12 @@ export function MyPage({ data, nav }) {
     debug: process.env.NEXT_PUBLIC_EXPERIMENTS_DEBUG,
     redirectFn: replace
   }
+
+  // Wrap the DotcmsLayout with the withExperiments HOC
+  const DotLayoutWithExperiments = withExperiments(DotcmsLayout);
```

Comment:
This should only happens if experiments ENV vars are set.

Reply:
Take a look now. I made a few changes there and added comments.